test/Driver/sanitizers.swift

```diff
@@ -64,5 +87,20 @@
 
 // TSAN: -rpath @executable_path
 
+// UBSAN: swift
+// UBSAN: -sanitize=undefined
+
+// UBSAN_OSX: lib/swift/clang/lib/darwin/libclang_rt.ubsan_osx_dynamic.dylib
+// UBSAN_IOSSIM: lib/swift/clang/lib/darwin/libclang_rt.ubsan_iossim_dynamic.dylib
+// UBSAN_IOS: lib/swift/clang/lib/darwin/libclang_rt.ubsan_ios_dynamic.dylib
+// UBSAN_tvOS_SIM: lib/swift/clang/lib/darwin/libclang_rt.ubsan_tvossim_dynamic.dylib
+// UBSAN_tvOS: lib/swift/clang/lib/darwin/libclang_rt.ubsan_tvos_dynamic.dylib
+// UBSAN_watchOS_SIM: lib/swift/clang/lib/darwin/libclang_rt.ubsan_watchossim_dynamic.dylib
+// UBSAN_watchOS: lib/swift/clang/lib/darwin/libclang_rt.ubsan_watchos_dynamic.dylib
+// UBSAN_LINUX: lib/swift/clang/lib/linux/libclang_rt.ubsan-x86_64.a
+// UBSAN_WINDOWS: lib/swift/clang/lib/windows/clang_rt.ubsan_standalone-x86_64.lib
```

Comment:
This lib name is based off @troughton earlier comment but I believe the logic in the windows driver will look for `clang_rt.ubsan-x86_64.lib` so this might require an update

Reply:
Ah, you should be able to run the tests locally to find out! docs/Testing.md has information on how to run just the one test.